datalad/metadata/aggregate.py

```diff
@@ -157,16 +158,19 @@ def _within_metadata_store(ds, guess_native_type, metapath):
         ds,
         guess_type=guess_native_type,
         ignore_subdatasets=True,
-        ignore_cache=True)
+        from_native=True)
     # strip git-based version info from the meta data that is cached
     # in the dataset itself -- this will be outdated the second we
     # commit below
+    # NOTE next line should not need protection, as we only process registered
+    # subdatasets
+    ds_identifier = ds.repo.repo.head.commit.hexsha
     for m in meta:
-        if not is_implicit_metadata(m):
+        if not m.get('@id', None) == ds_identifier:
             continue
-        for prop in ('dcterms:modified', 'version'):
-            if prop in m:
-                del m[prop]
+        m['@id'] = 'THISDATASET!'
```

Comment:
pardon me -- what does it mean? i.e. why **this super dataset** doesn't have its own ID?
```shell
(git)hopa:/tmp/datasets-dev.datalad.org[master].datalad/meta
$> git grep THISDA
crcns/meta.json:    "isPartOf": "THISDATASET!",
dicoms/meta.json:    "isPartOf": "THISDATASET!",
indi/meta.json:    "isPartOf": "THISDATASET!",
kaggle/meta.json:    "isPartOf": "THISDATASET!",
labs/meta.json:    "isPartOf": "THISDATASET!",
meta.json:    "@id": "THISDATASET!",
neurovault/meta.json:    "isPartOf": "THISDATASET!",
nidm/meta.json:    "isPartOf": "THISDATASET!",
openfmri/meta.json:    "isPartOf": "THISDATASET!",
```